packages/runtime-core/src/renderer.ts

```diff
@@ -1098,7 +1098,9 @@ function baseCreateRenderer(
         dynamicChildren &&
         // #2715 the previous fragment could've been a BAILed one as a result
         // of renderSlot() with no valid children
-        n1.dynamicChildren
+        n1.dynamicChildren &&
+        // #9200 in some case stable fragment in deep unstable slot
+        n1?.children?.length === n2?.children?.length
```

Comment:
We should avoid using optional chaining https://github.com/vuejs/core/pull/4882#issuecomment-957064008

Reply:
removed optional chaining